src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,6 +129,47 @@ public void testKeyEncryption() throws InvalidParametersException {
     }
 
     @Test
+    public void testDefaultAesCipherParams() throws Exception {
+        //plaintext is less than 1 block long
+        String plaintextLength124 = StringHelper.generateStringOfLength(31, 'A');
+        assertEncryptionUsesExpectedDefaultParams(plaintextLength124);
+
+        //plaintext is equal to 1 block
+        String plaintextLength128 = StringHelper.generateStringOfLength(32, 'A');
+        assertEncryptionUsesExpectedDefaultParams(plaintextLength128);
+
+        //plaintext is more than 1 block long
+        String plaintextLength136 = plaintextLength128 + StringHelper.generateStringOfLength(2, 'A');
```

Comment:
The other two cases are `31` and `32`; shouldn't this add up to `33`? i.e. use `1` instead of `2`